tools/publish_http.py

```diff
@@ -228,8 +228,8 @@ def main(argv):
         logger.info('Install your package using the following command:')
     else:
         logger.info('Install your package using the following commands:')
-        logger.info('dcos package repo add --index=0 {}-aws {}'.format(self._pkg_name, universe_url))
-    logger.info('dcos package install {}'.format(self._pkg_name))
+        logger.info('dcos package repo add --index=0 {}-aws {}'.format(package_name, universe_url))
+    logger.info('dcos package install {}'.format(package_name))
```

Comment:
For consistency, should we also put this "add-package-and-install" log message for the `aws` script?